index.html

```diff
@@ -13,3 +13,4 @@ <h1 class="greeting">Hello Microverse!</h1>
     </header>
   </body>
 </html>
+<!-- For code review -->
```

Comment:
- Please add a description for the PR you can edit it by pressing the three dots icon then pressing edit 
![image](https://user-images.githubusercontent.com/24703171/153063912-a4a53864-b19a-459e-af5b-94096f9c0475.png)

Reply:
@gemmen29 Thank you for your observations  💪